Assets/MixedRealityToolkit-Preview/LightEstimation/Scripts/Editor/LightCaptureEditor.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.LightEstimation
+{
+	[CustomEditor(typeof(LightCapture))]
+	public class LightCaptureEditor : Editor
+	{
+		SerializedProperty mapResolution;
+		SerializedProperty singleStampOnly;
+		SerializedProperty stampFovMultiplier;
+		SerializedProperty stampExpireDistance;
+		SerializedProperty useDirectionalLight;
+		SerializedProperty maxLightColorSaturation;
+		SerializedProperty lightAngleAdjustPerSecond;
+		SerializedProperty cameraOrientation;
+		SerializedProperty probe;
+		SerializedProperty directionalLight;
+
+		private void OnEnable()
+		{
+			mapResolution       = serializedObject.FindProperty("mapResolution");
+			singleStampOnly     = serializedObject.FindProperty("singleStampOnly");
+			stampFovMultiplier  = serializedObject.FindProperty("stampFovMultiplier");
+			stampExpireDistance = serializedObject.FindProperty("stampExpireDistance");
+			useDirectionalLight = serializedObject.FindProperty("useDirectionalLight");
+			maxLightColorSaturation = serializedObject.FindProperty("maxLightColorSaturation");
+			lightAngleAdjustPerSecond = serializedObject.FindProperty("lightAngleAdjustPerSecond");
+			cameraOrientation   = serializedObject.FindProperty("cameraOrientation");
+			probe               = serializedObject.FindProperty("probe");
+			directionalLight    = serializedObject.FindProperty("directionalLight");
+		}
+		public override void OnInspectorGUI()
+		{
+			EditorGUILayout.LabelField("Quality Presets", EditorStyles.boldLabel);
+			int preset = EditorGUILayout.Popup("Apply preset:", 0, new string[] { "...", "Super Low", "Fast", "High", "Very High"});
+			if (preset == 1)  // Low
```

Comment:
Nit: I found this UI somewhat confusing since my selection always went back to "..." It might be cool if all the settings are grayed out if SuperLow/Low/Medium/Normal/High is selected and then add another settings called "Custom" that un-grays the options and allows the user to pick whatever.

Reply:
This is something I would like to do, but don't know of a good way to do it yet! I'm not a fan of brute checking it, but I'm certain there's a more elegant way, so I'll definitely be keeping this in mind for future updates.